src/api/app/models/workflow/step.rb

```diff
@@ -14,6 +16,7 @@ def call(_options)
     raise AbstractMethodCalled
   end
 
+  # FIXME: Rename this function because its clashing with names below
```

Comment:
I don't agree with this comment. If we turn `workflow_repositories` into a query object, this wouldn't be an issue anymore.